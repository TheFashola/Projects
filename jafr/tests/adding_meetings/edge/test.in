2

 meet batman
 
22/03/23
 
12:00
n