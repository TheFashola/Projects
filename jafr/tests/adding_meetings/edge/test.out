Just a friendly reminder! You have these tasks to finish today.

These tasks need to be finished in the next three days!
- Budget Approval for Next Quarter by 04/09/23
- Pick Up Birthday Gift for Barbara by 04/09/23
- Draft Speech for Town Hall by 05/09/23
- Review Neighborhood Watch Applications by 05/09/23

You have the following meetings today!
- Task Force Briefing on Two-Face at 10:00

You have the following meetings scheduled over the next week!
- Town Hall Meeting on Public Safety on 04/09/23 at 18:00
- Meeting with Mayor on 04/09/23 at 13:00

What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
2
Please enter a meeting description: 

invalid descriptionPlease enter a meeting description: meet batman

Please enter a date: 

invalid datePlease enter a date:22/03/23

Please enter a time: 

invalid timePlease enter a time:12:00

Ok, I have added  meet batman on 22/03/23 at 12:00.
Would you like to share this meeting? [y/n]: n
What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit